_includes/tutorials/aggregating-average/kstreams/code/build.gradle

```diff
@@ -0,0 +1,64 @@
+buildscript {
+    repositories {
+        jcenter()
+    }
+    dependencies {
+        classpath "com.commercehub.gradle.plugin:gradle-avro-plugin:0.15.1"
+        classpath "com.github.jengelman.gradle.plugins:shadow:4.0.2"
+    }
+}
+
+plugins {
+    id "java"
+    id "com.google.cloud.tools.jib" version "1.1.1"
+    id "idea"
+    id "eclipse"
+}
+
+sourceCompatibility = "1.8"
+targetCompatibility = "1.8"
+version = "0.0.1"
+
+repositories {
+    jcenter()
+    maven { url 'http://packages.confluent.io/maven/' }
+}
+
+apply plugin: "com.commercehub.gradle.plugin.avro"
+apply plugin: "com.github.johnrengelman.shadow"
+
+dependencies {
+    compile "org.apache.avro:avro:1.8.2"
+    implementation "org.slf4j:slf4j-simple:1.7.26"
+    implementation "org.apache.kafka:kafka-streams:2.4.0"
+    implementation "io.confluent:kafka-streams-avro-serde:5.4.0"
+
+    compile group: 'com.typesafe', name: 'config', version: '1.4.0'
+    testCompile "org.apache.kafka:kafka-streams-test-utils:2.4.0"
+    testCompile "junit:junit:4.12"
+    testImplementation 'org.hamcrest:hamcrest:2.2'
+
+    testCompileOnly "org.projectlombok:lombok:1.18.12"
+    testAnnotationProcessor "org.projectlombok:lombok:1.18.12"
+}
+
+test {
+    testLogging {
+        outputs.upToDateWhen { false }
+        showStandardStreams = true
+        exceptionFormat = "full"
+    }
+}
+
+jar {
+  manifest {
+    attributes(
+      "Class-Path": configurations.compile.collect { it.getName() }.join(" "),
+      "Main-Class": "io.confluent.developer.RunningAverage.java"
```

Comment:
`io.confluent.developer.RunningAverage.java` should just be `io.confluent.developer.RunningAverage` .  I had an error in the `gen_project` script that I've fixed now.

Reply:
Thanks!